nubison_model/Service.py

```diff
@@ -67,10 +63,33 @@ def build_inference_service(
     mlflow_model_uri = mlflow_model_uri or getenv(ENV_VAR_MLFLOW_MODEL_URI) or ""
 
     num_workers = int(getenv(ENV_VAR_NUM_WORKERS) or DEFAULT_NUM_WORKERS)
+    shared_artifacts_path = "/tmp/mlflow_artifacts"
+
+    def download_mlflow_artifacts():
+        """Download MLflow artifacts to shared directory before service creation."""
+        from mlflow.artifacts import download_artifacts
+
+        if not mlflow_tracking_uri:
+            raise RuntimeError("MLflow tracking URI is not set")
+        if not mlflow_model_uri:
+            raise RuntimeError("MLflow model URI is not set")
+
+        try:
+            set_tracking_uri(mlflow_tracking_uri)
+            makedirs(shared_artifacts_path, exist_ok=True)
```

Comment:
It seems better to use something like TemporaryDirectory to create a temporary directory for storing artifacts.
It can automatically create an appropriate temporary directory path depending on the OS.